content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -9,7 +9,6 @@ weight: 20
 <!-- overview -->
 This page provides an overview of DNS support by Kubernetes.
 
-
```

Comment:
nit: Does it make sense to remove line 10 (This page provides an overview...)?

Reply:
I'll write an actually helpful overview :)